apps/meteor/app/custom-oauth/server/custom_oauth_server.js

```diff
@@ -123,35 +123,32 @@ export class CustomOAuth {
 
 		// Only send clientID / secret once on header or payload.
 		if (this.tokenSentVia === 'header') {
-			allOptions.auth = `${config.clientId}:${OAuth.openSecret(config.secret)}`;
+			allOptions.headers.Authorization = Buffer.from(`${config.clientId}:${OAuth.openSecret(config.secret)}`).toString('base64');
```

Comment:
Aren't we forgetting to concatenate a `Basic ${base64}` here together with the base64 string, informing which type of auth we are sending?